### ISSUE TITLE
Exit Multi-Line Flashcard Header

### ISSUE THREAD

**Issue by nattheninja** [remnoteio/remnote-issues#1548]
**Describe the bug**
When you create a multi-line flashcard, you used to be able to exit the card by entering shift + tab and then tabbing back to the same level of the multi-line flashcard. A previous workaround that allowed you to exit the multiline flashcard was hitting the enter key to the non-multiline rem, and then entering /p for no headers to remove the bolded line that indicated that a rem was part of a multiline flashcard.

**To Reproduce**
1. Create a multi-line flashcard with one or two sub rems as answers
2. Attempt to exit the multi-line flashcard creation by clicking shift tab to go one level up, then tab back in
3. If Step 2 does not work,  attempt to use /p on the same level as multi-line flashcard answer to exit flashcard
4. Any rems below the multi-line flashcard remain highlighted, indicating that they will be treated as part of the flashcard when not intended to be

**Expected behavior**
Hitting Shift+Tab and tabbing back in to the same level of a multi-line flashcard should not include the new rems in the multi-line flashcard answers. Similarly, hitting /p should exit the multi-line flashcard creation as well.

**Screenshots**
![image](https://user-images.githubusercontent.com/101621442/158317575-4277fcb4-cfc2-43d8-9a45-2b353ea9e8a7.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser [e.g. chrome, safari]: Chrome
 - Version [e.g. 22]: 1.7.5 (Remnote) 

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6] N/A
 - OS: [e.g. iOS8.1] N/A
 - Browser [e.g. stock browser, safari] N/A
 - Version [e.g. 22] N/A

**Additional context**
Workaround for /p was functional prior to 1.7.3 update.

**Comment by TaQuangKhoi** [comment]
You can make a feature request in http://feedback.remnote.com/ ..

**Comment by nattheninja** [comment]
Okay, just reported the bug. I don't think it is a feature request because the intended functionality is demonstrated here: https://www.youtube.com/watch?v=YIHoH2ySLrU&t=190s

**Comment by TaQuangKhoi** [comment]
> 

Yeah, I see.. There used to be a feature like that.. Hope this feature will be back soon after your report..
